x-pack/plugin/data-frame/qa/single-node-tests/build.gradle

```diff
@@ -2,17 +2,50 @@ apply plugin: 'elasticsearch.standalone-rest-test'
 apply plugin: 'elasticsearch.rest-test'
 
 dependencies {
-  testCompile "org.elasticsearch.plugin:x-pack-core:${version}"
+  testCompile project(path: xpackModule('core'), configuration: 'default')
+  testCompile project(path: xpackModule('core'), configuration: 'testArtifacts')
   testCompile project(path: xpackModule('data-frame'), configuration: 'runtime')
 }
 
+integTestRunner {
```

Comment:
I know very little in how our build system works. I added fields from what is in the ml native integration tests until the tests started executing. If anybody wants to give me a run down, and show me how most of these changes are unnecessary for my native client tests, please do :).

Reply:
I have since paired down to only what is necessary. Next commit will have fewer changes to the build file